### ISSUE TITLE
Cmd+J on MacOS to hide/show terminal shows broken outline

### ISSUE THREAD

**Issue by cmdrkeene** [microsoft/vscode#95267]
Issue Type: <b>Bug</b>

Cmd+J on MacOS to hide/show terminal/bottom-tray shows broken outline unlike Toggle Terminal shortcut Ctrl+`

Changing text size up or down corrects outline of terminal

VS Code version: Code 1.44.0 (2aae1f26c72891c399f860409176fe435a154b13, 2020-04-08T08:23:56.137Z)
OS version: Darwin x64 18.7.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8569U CPU @ 2.80GHz (8 x 2800)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off_ok<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|3, 3, 3|
|Memory (System)|16.00GB (0.54GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (6)</summary>

Extension|Author (truncated)|Version
---|---|---
pgformatter|bra|1.14.0
prettier-vscode|esb|4.4.0
ruby|reb|0.27.0
vscode-scss-formatter|sib|2.0.0
simple-ruby-erb|vor|0.2.1
vscode-ruby|win|0.27.0

</details>

Screenshot of broken terminal outline:

<img width="336" alt="Screen Shot 2020-04-14 at 4 13 21 PM" src="https://user-images.githubusercontent.com/22959/79269704-e0b7da80-7e6a-11ea-97f9-98e9d8707253.png">

**Comment by meganrogge** [comment]
I cannot reproduce with the above steps so I'm closing this, but if you can still reproduce, please let me know. A [gif](https://gifcap.dev/) would be helpful.
